From 7cecc9138b89e1946e3e515727bb69b2ab119806 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Thu, 26 May 2022 07:11:37 +0100
Subject: [PATCH] add missing <cstdint> header inclusion (gcc-13 support)

Without the change ade build fails on this week's gcc-13 snapshot as:

    [ 77%] Building CXX object sources/ade/CMakeFiles/ade.dir/source/topological_sort.cpp.o
    In file included from ade/sources/ade/include/ade/passes/topological_sort.hpp:17,
                     from ade/sources/ade/source/topological_sort.cpp:7:
    ade/sources/ade/include/ade/typed_graph.hpp:101:10: error: 'uintptr_t' in namespace 'std' does not name a type
      101 |     std::uintptr_t m_srcGraph;
          |          ^~~~~~~~~

Upstream-Status: Backport
Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 sources/ade/include/ade/typed_graph.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sources/ade/include/ade/typed_graph.hpp b/sources/ade/include/ade/typed_graph.hpp
index d9676e6..8a27578 100644
--- a/sources/ade/include/ade/typed_graph.hpp
+++ b/sources/ade/include/ade/typed_graph.hpp
@@ -9,6 +9,7 @@
 #ifndef ADE_TYPED_GRAPH_HPP
 #define ADE_TYPED_GRAPH_HPP
 
+#include <cstdint>
 #include <unordered_set>
 #include <string>
 
